.github/workflows/lint.yaml

```diff
@@ -22,3 +22,7 @@ jobs:
     - name: Lint with flake8
       run: |
         flake8 $(git ls-files 'sriov/common/*.py') --count --ignore=E203 --max-complexity=15 --max-line-length=88 --show-source --statistics
+        flake8 $(git ls-files 'sriov/tests/SR_IOV_macAddress_DPDK/*.py') --count --ignore=E203 --max-complexity=15 --max-line-length=88 --show-source --statistics
```

Comment:
Just to be clear, eventually we want to have a just single line that can apply to all py files in the tests folder; we temporarily add per test case folder until we have updated all test cases. right?

Reply:
Correct. As I mentioned above, we can consolidate these later once all tests are ready for linter.